css/auth.css

```diff
@@ -0,0 +1,11 @@
+.auth-container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    max-width: 30rem;
+    background-color: blanchedalmond;
```

Comment:
- use var here for colors

Reply:
This color is being used only at one place, hence I did not create a variable